src/OpenTelemetry.Api/Context/Propagation/TraceContextFormatActivity.cs

```diff
@@ -42,6 +42,25 @@ public class TraceContextFormatActivity : ITextFormatActivity
         /// <inheritdoc/>
         public ISet<string> Fields => new HashSet<string> { TraceState, TraceParent };
 
+        /// <inheritdoc/>
+        public bool IsInjected<T>(T carrier, Func<T, string, IEnumerable<string>> getter)
+        {
+            try
+            {
+                var traceparentCollection = getter(carrier, TraceParent);
+
+                // There must be a single traceparent
+                return traceparentCollection != null && traceparentCollection.Count() == 1;
```

Comment:
so IsInjected will return false if there are two of them?

Reply:
looks counterintuitive

Reply:
I mirrored what is done in the `Extract` logic 🤷‍♂️ Easy enough to change it to return `true` if at least one is found. Let me know if you want to change it just here, in both spots, leave as-it, or whatever.